src/main/java/htsjdk/samtools/fastq/FastqReader.java

```diff
@@ -41,6 +41,22 @@
  * directly.  It is provided so that this class can be used in Java for-each loop.
  */
 public class FastqReader implements Iterator<FastqRecord>, Iterable<FastqRecord>, Closeable {
+    /** Enum of the types of lines we see in Fastq. */
+    protected enum LineType {
+        SequenceHeader("Sequence Header"),
+        SequenceLine("Sequence Line"),
+        QualityHeader("Quality Header"),
+        QualityLine("Quality Line");
+
+        private String printable;
+
+        LineType(String printable) {
+            this.printable = printable;
+        }
+
+        @Override public String toString() { return this.printable; }
```

Comment:
Seeing the extra requests for tests like this seems like a burden to the submitter, even if the test guards against a regression.  It's a fine line, but I'd be fine without the test.  Just my 2 cents.